Token.js

```diff
@@ -49,6 +52,14 @@ class Token {
 		if (typeof options.conditions == "undefined") {
 			this.options.conditions = [];
 		}
+
+		// Store scene boundaries when token is created
```

Comment:
I would convert this to a function instead of defining it at the time of creation. It's _probably unlikely_ that the scene size would change, but not impossible.

Reply:
Should also probably live in the sceneshandler. Feels a bit weird that the token class itself knows the boundaries of the scene. Same for the scene_move_grid_padding_multiplier